src/lib/temple/front/balance.ts

```diff
@@ -18,8 +25,19 @@ type UseBalanceOptions = {
 };
 
 export function useBalance(assetSlug: string, address: string, opts: UseBalanceOptions = {}) {
```

Comment:
Sorry, this approach won't work as expected because there is no logic for suspense while balances are loading from TZKT. By the way, `balancesAreKnownToStore` is `true` at a part of that time

Reply:
However, the code has been rewritten to rely on type definitions a bit more